src/core/meshcop/tcat_agent.cpp

```diff
@@ -389,6 +389,7 @@ Error TcatAgent::HandleSingleTlv(const Message &aIncomingMessage, Message &aOutg
         {
         case kTlvDisconnect:
             error = kErrorAbort;
+            response = true; // true - to avoid response-with-status being sent.
```

Comment:
nit: to fix the pretty check
```suggestion
            error    = kErrorAbort;
            response = true; // true - to avoid response-with-status being sent.
```

Reply:
nit fixed now!